bin/templates/cordova/Api.js

```diff
@@ -51,7 +51,7 @@ function setupEvents (externalEventEmitter) {
 
 class Api {
     constructor (platform, platformRootDir, events) {
-        this.platform = 'electron';
+        this.platform = 'cordova-electron';
```

Comment:
```suggestion
        this.platform = 'electron';
```

Can you explain why it needs to change to `cordova-electron`? 

I believe for all other platforms it does not include `cordova-`, for example, they are: `osx`, `ios`, `android`, `browser`.

I would like to keep it consistent but would like to know why it doesn't work this way for the electron platform.